tests/infrastructure/downward-metrics.go

```diff
@@ -93,4 +99,95 @@ var _ = DescribeInfra("downwardMetrics", func() {
 		Expect(found).To(BeTrue())
 		Expect(metrics.Metrics[j].Value).To(Equal("1"))
 	})
+
+	DescribeTable("should not start a VMI if the feature is disabled until it is re-enabled", func(via libvmi.Option) {
+		By("disabling downwardMetrics feature")
+		tests.DisableDownwardMetrics(virtClient)
+
+		DeferCleanup(func() {
+			By("re-enabling the downwardMetrics feature")
+			tests.EnableDownwardMetrics(virtClient)
```

Comment:
This will leave the feature enabled for other tests?

Reply:
It should, yes.